src/Behat/Config/Profile.php

```diff
@@ -244,18 +250,34 @@ private function addUnusedDefinitionsToExpr(Expr &$expr): void
 
     private function addPathOptionsToExpr(Expr &$expr): void
     {
-        if (!isset($this->settings[self::PATH_OPTIONS_SETTING][self::PRINT_ABSOLUTE_PATHS_SETTING])) {
-            return;
+        $args = [
```

Comment:
When calling the `addMethodCall` function later we need to have values for all parameters as they are needed in the function that checks the default values, so we start with the default values and modify them if any path option setting is found

Reply:
This feels a little verbose / hard to follow to me (especially if we add more path options settings in future). Is there a way we could simplify it? Maybe with something like a map of setting name => parameter name and then a foreach over them?

Reply:
Let me know if this looks better now

Reply:
Thanks, yes I find that much clearer